src/logging.h

```diff
@@ -19,15 +21,7 @@ static const bool DEFAULT_LOGIPS        = false;
 static const bool DEFAULT_LOGTIMESTAMPS = true;
 extern const char * const DEFAULT_DEBUGLOGFILE;
 
-extern bool fPrintToConsole;
-extern bool fPrintToDebugLog;
-
-extern bool fLogTimestamps;
-extern bool fLogTimeMicros;
 extern bool fLogIPs;
```

Comment:
In commit _util: Establish global logger object._ : any reason to exclude `fLogIPs` from `g_logger`?

Reply:
It's not actually used by the logger, it's just used in the net code. Arguably it should live there instead.